src/pytest_plugins/pytest_hive/pytest_hive.py

```diff
@@ -107,11 +107,17 @@ def hive_test(request, test_suite: HiveTestSuite):
     """
     Propagate the pytest test case and its result to the hive server.
     """
+    try:
+        fixture_description = request.getfixturevalue("fixture_description")
+    except pytest.FixtureLookupError:
+        pytest.exit(
+            "Error: The 'fixture_description' fixture has not been defined by the simulator "
+            "or pytest plugin using this plugin!"
+        )
     test_parameter_string = request.node.nodeid.split("[")[-1].rstrip("]")  # test fixture name
```

Comment:
```suggestion
    test_parameter_string = request.node.nodeid  # test fixture name
```

I'm not sure if this is the correct suggestion, but for hive pyspec we have the full test name:
<img width="880" alt="image" src="https://github.com/ethereum/execution-spec-tests/assets/60348173/2e45ad8d-8c3e-44fd-8720-51cdd5d40baf">

Currently in consume rlp we have:
<img width="753" alt="image" src="https://github.com/ethereum/execution-spec-tests/assets/60348173/af9fc9de-6a2b-4652-ae86-460d7f839332">
for the same test - and from that alone I have no idea what the test is 🥲

Reply:
Ah ok, yes, but that's an existing issue that i must have chose to ignore - it's not directly related to what we're adding n this PR :-) Will take a look!

But with this PR, you will get the fixture **ID** in the "Description" for easy copy-paste. I.e., you can copy this ID pass it to `fill`. 

Perhaps we can add the consume test ID too, so that you pass it directly to `consume` - this is probably more helpful.

I think the goal would be to provide versions of all components and two command-lines in the description to allow easy local re-execution via either:
1. `consume rlp` (or `consume engine`).
2. `./hive`.

Reply:
> Perhaps we can add the consume test ID too, so that you pass it directly to consume - this is probably more helpful.

💯

Reply:
Would love a backwards compatible `-k` option for hive actually. Where running hive with `--sim.limit "/eip2935"` does the same as `consume -k eip2935` :D